vuu-ui/packages/vuu-filters/src/filter-clause/ExpandoCombobox.tsx

```diff
@@ -52,8 +54,8 @@ export const ExpandoCombobox = forwardRef(function ExpandoCombobox<
   const { itemToString = defaultToString } = props;
   const initialValue = useRef(value);
 
-  const itemsToString = useCallback(
-    (items: Item[]) => {
+  const itemsToString = useCallback<<I = Item>(items: I[]) => string>(
+    (items) => {
       const [first, ...rest] = items;
       if (rest.length) {
         return `${itemToString(first)} + ${rest.length}`;
```

Comment:
We might be able to do something like:

`const itemsToString = useCallback(
    (items: Array<Item> | Array<string>) => {
      const [first, ...rest] = items;
      const firstAsString =
        typeof first === "string" ? first : itemToString(first);
      if (rest.length) {
        return `${firstAsString} + ${rest.length}`;
      } else {
        return firstAsString;
      }
    },
    [itemToString]
  );`
  
  this way we can keep the type to `itemToString?: (i: Item) => string`, and similarly avoid type casting such as `column as ColumnDescriptor`
  
  What do you think? If you want we can do this in a separate follow-up change.

Reply:
I like it, lets do it after I merge this one.